docs/ECOS.md

```diff
@@ -0,0 +1,141 @@
+# Edge Containers
+
+## Introduction
+
+While EVE provides a lot of sophisticated device management functionality, at the end of the day its success depends on a diverse and growing ecosystem of applications that it can support. EVE is *not* a traditional operating system in that it doesn't support applications represented as executable files and [POSIX processes](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_295). Instead EVE expects its applications to be:
+
+* [Traditional Virtual Machines](https://en.wikipedia.org/wiki/Virtual_machine)
+* [Unikernels](http://unikernel.org/)
+* [Docker/OCI container](https://www.opencontainers.org)
+
+Edge Container is a novel concept that allows an effective packaging and lifecycle management of all these types of applications and it consists of two parts:
+
+* Edge Container Image (ECI) - an self-contained, binary representation of an Edge Container
+* Edge Container Object (ECO) - a live copy of an Edge Container
```

Comment:
"Object" is a bit of a challenging word. I think "Image" makes a lot of sense, and is used regularly for things like containers and VMs and disks, because people get it. On the other hand, I get why you want to use it. For those other cases, we always had "Virtual Machine" (actual running machine) / "Virtual Machine Image" (binary rep that can be run). Same for "Container" (actual running container) / "Container Image" (binary rep that can be run). 

Conversely, since that appears to be the (not so great) pattern, maybe we should just stick with it, and have "Edge Container" (actual running EC) / "Edge Container Image" (binary rep that can be run). I don't love it (as I don't love the others' nomenclature), but it has stuck.

Reply:
Whether ECO = "Edge Container Object" or ECO = "Edge COntainer", I really like the ECO/ECI shorthand.

Reply:
Thanks for the feedback -- lets think about this a bit more since I honestly feel that this bleeds into a marketing/branding category (and I don't say this lightly -- this kind of marketing is actually important).

For now, lets just go with what's there and see how it resonates.